module-system/sov-modules-api/src/capabilities.rs

```diff
@@ -0,0 +1,35 @@
+#![deny(missing_docs)]
+
+//! Rollup capabilities rust module defines rollup specific capabilities,
+//! which can be implemented by different module
```

Comment:
nit: this comment is a little hard to parse. How about

```
//! The rollup capabilities module defines "capabilities" that rollups must
//! provide if they wish to use the standard app template. If you don't 
//! want to provide these capabilites, you can bypass the Sovereign module-system completely and write a 
//! state transition function from scratch. See here for docs (TODO: insert link)
```

Reply:
Yes, it needs some love.

I decided to always add `#![deny(missing_docs)]` to any new rust module I write, so debt is not accumulated.